ci-operator/step-registry/upi/conf/vsphere/ova/upi-conf-vsphere-ova-commands.sh

```diff
@@ -18,8 +18,11 @@ fcos_json_file="${openshift_install_path}/fcos.json"
 if [[ -f "$fcos_json_file" ]]; then
     image_json_file=$fcos_json_file
 fi
-
-ova_url="$(jq -r '.baseURI + .images["vmware"].path' $image_json_file)"
+if test -f "${openshift_install_path}/rhcos-stream.json"; then
```

Comment:
Hmm while we have the patient open it's tempting to call this `coreos-stream.json` since it applies to FCOS too.